src/codegen.cpp

```diff
@@ -1363,6 +1363,7 @@ extern "C" {
         /* gnu_pubnames */ 1,
 #endif
         /* debug_info_kind */ (int) DICompileUnit::DebugEmissionKind::FullDebug,
+        /* debug_line_info */ 1,
```

Comment:
```suggestion
        /* debug_info_level */ static_cast<int>(jl_options.debug_level),
```

(I realize this is not valid code, and I don't really know how you would go about fixing it. Maybe making it `-1` as a sentinal value?)

Reply:
I can look for places where we instantiate the default and set it to the correct thing. Or given that we don't mark this as const maybe we can modify it during init?